src/core/public/index.scss

```diff
@@ -9,3 +9,4 @@
 
 @import './chrome/index';
 @import './overlays/index';
+@import '../../plugins/index';
```

Comment:
Is this the way to hook up plugin's SASS from core?

Would be nice to have a comment